pkg/operator/targetconfigcontroller/targetconfigcontroller.go

```diff
@@ -97,7 +95,7 @@ func NewTargetConfigController(
 	c.cachesSync = append(c.cachesSync, configInformer.Config().V1().Infrastructures().Informer().HasSynced)
 
 	configInformer.Config().V1().FeatureGates().Informer().AddEventHandler(c.eventHandler())
-	c.cachesSync = append(c.cachesSync, configInformer.Config().V1().FeatureGates().Informer().HasSynced)
+	c.cachesSync = append(c.cachesSync, configInformer.Config().V1().FeatureGates().Informer().HasSynced, configInformer.Config().V1().Schedulers().Informer().HasSynced)
```

Comment:
it is always a good idea to wait for caches to sync before using an informer

Reply:
if this use controller factory you get it for free and never forget :-)

Reply:
true, at some point, we should refactor and use the new factory.